File: app/src/main/java/com/king/superdemo/utils/CommonUtil.java
package com.king.superdemo.utils;

import android.content.Context;

public class CommonUtil {

    public static int getStatusBarHeight(Context context) {
        int statusBarHeight = 0;
        int resID = context.getResources().getIdentifier("status_bar_height", "dimen", "android");
        if (resID > 0)
            statusBarHeight = context.getResources().getDimensionPixelSize(resID);
        return statusBarHeight;
    }
}

File: app/src/main/java/com/king/superdemo/utils/CommonUtil.kt
package com.king.superdemo.utils

import android.content.Context

object CommonUtil {
    fun getStatusBarHeight(context: Context): Int {
        var statusBarHeight = 0
        val resID = context.resources.getIdentifier("status_bar_height", "dimen", "android")
        if (resID > 0) statusBarHeight = context.resources.getDimensionPixelSize(resID)
        return statusBarHeight
    }
}